devtools/src/main/scala/kalix/devtools/impl/DockerComposeUtils.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.devtools.impl
+
+import java.nio.file.Files
+import java.nio.file.Paths
+
+import scala.sys.process._
+
+object DockerComposeUtils {
```

Comment:
Moved logic to this utility class and in dev-tools in order to use it from inside the maven plugin as well.